OOPS Concepts.txt

```diff
@@ -0,0 +1,21 @@
+Questions and Answers:-
+
+1)Real-world objects contain ___ and ___.
+Ans) State and Behaviour.
+2)A software object's state is stored in ___.
+Ans) Fields
+3)A software object's behavior is exposed through ___.
+Ans) Methods
+4)Hiding internal data from the outside world, and accessing it only through publicly exposed methods is known as data ___.
+Ans) Abstraction
+5)A blueprint for a software object is called a ___.
+Ans) Class
+6)Common behavior can be defined in a ___ and inherited into a ___ using the ___ keyword.
+Ans) Parent Class, Child Class, Extends

```

Comment:
The answer is super class,sub class and extends

Reply:
Yes sir